app/views/plots/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>Plots</h1>
+<ul>
+  <% @plots.each do |plot| %>
+    <li id = "plot-<%= plot.id %>">
+      Plot Number: <%= plot.number %>
+      <ul>
+        <% plot.plants.each do |plant| %>
+        <div id = "plant-<%= plant.id %>">
+          <li><%= plant.name %></li>
+          <li><%= link_to "Remove", plot_plant_path(plot,plant), method: :delete %>
```

Comment:
Just a reminder that Turbo can sometimes cause problems with link_tos that use the :delete method. You usually have to add `data: {turbo_method: :delete}`. Your tests might pass but when running the server, if you see funky behavior (or the link just routes as a GET no matter what instead of a DELETE) then usually Turbo is the problem. 

Instead, you can also just choose to use a button! Turbo doesn't mess with buttons